### PR TITLE
Completed task 4 list all articles

### PR DESCRIPTION
**Pull request by megclaycla**

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  const [articles, setArticles] = useState([])
+
+
+  return (
+    <UserContext.Provider value={{loggedInUser:loggedInUser}}>
+      <Header />
+      <Routes>
+        <Route path='/' element={<Home />} />
+        <Route path='/topics' element={<TopicsList articles={articles} setArticles={setArticles} />} />
+        <Route path='/articles' element={<ArticlesList articles={articles} setArticles={setArticles} />} />
```
Comment (nc-saima-khanom) [inline src/components/App/App.jsx:21]: Is there a reason we have declared the articles state so high up in the app when the articles list component is able to make the request and populate this state based on the categories - does this eliminate the need for the topics component?

File: src/components/App/App.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { Routes, Route } from 'react-router-dom'
+import TopicsList from '../TopicsList/TopicsList'
+import ArticlesList from '../ArticlesList/ArticlesList'
+import UsersList from '../UsersList/UsersList'
+import UserContext from '../../contexts/User'
+
+function App() {
+  const [loggedInUser, setLoggedInUser] = useState({username: 'Nala', avatar_url: 'https://images.unsplash.com/photo-1611752554510-94b98a0e64ae?q=80&w=1470&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D'})
+  const [articles, setArticles] = useState([])
+
```
Comment (nc-saima-khanom) [inline src/components/App/App.jsx:13]: we should have a loading state for the article load

File: src/components/ArticleCard/ArticleCard.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Link } from "react-router-dom";
+import './ArticleCard.css'
+
+function ArticleCard(props) {
+    const { article } = props
+    return (
+        <section className="article-card">
+            <Link to={`/articles/${article.article_id}`}><h3>{article.title}</h3></Link>
+            <p>Author: {article.author}</p>
```
Comment (nc-saima-khanom) [inline src/components/ArticleCard/ArticleCard.jsx:9]: Consider what other info the user might want to see with regards to each article before they select it - e.g. comment count, votes, author, date created in a human readable format - see this article for more interesting ways of displaying information: https://www.refactoringui.com/previews/labels-are-a-last-resort

File: src/components/App/App.jsx (hunk 2 above)
Comment (megclaycla) [reply src/components/App/App.jsx:13]: Added this :)

File: src/components/ArticleCard/ArticleCard.jsx (hunk 3 above)
Comment (megclaycla) [reply src/components/ArticleCard/ArticleCard.jsx:9]: Thanks for all the feedback, Saima. I've added in more details on the list of all articles.
I'm unsure how to change the date to a readable format. Am i best making a separate component to convert the date using toDateStamp? Will change after the lecture :)

**Pull request merged**